ruby/student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
+    super(age, parent_permission, name)
```

Comment:
- [ ] Please edit the `parent_permission` parameter to be true as default like so: `parent_permission = true`